lua/terrortown/lang/en.lua

```diff
@@ -1622,3 +1622,200 @@ L.label_roles_credits_award_kill = "Credit reward amount for kill"
+L.help_karma = "Karma is used to reduce random killing. Players start with a certain amount of Karma, and lose it when they damage/kill team mates. The amount they lose is dependent on the Karma of the person they hurt or killed. Lower Karma reduces damage given."
+L.help_karma_strict = "If strict Karma is enabled, the damage penalty increases more quickly as Karma goes down. When it is off, the damage penalty is very low when people stay above 800. Enabling strict mode makes Karma play a larger role in discouraging any unnecessary kills, while disabling it results in a more “loose” game where Karma only hurts players who constantly teamkill."
+L.help_karma_max = "Setting the value of the max Karma above 1000 doesn't give a damage bonus to players with more that 1000 Karma. It can be used as a Karma buffer."
+L.help_karma_ratio = "The ratio of the damage that is used to compute how much of the victim's Karma is subtracted from the attacker's if both are in the same team. If a team kill happens, a further penalty is applied."
+L.help_karma_traitordmg_ratio = "The ratio of the damage that is used to compute how much of the victim's Karma is subtracted from the attacker's if both are in different teams. If a team kill happens, a further bonus is applied."
+L.help_karma_bonus = "There are also two different passive ways to gain Karma during a round. First a round heal is applied to every player. Then a secondary clean bonus is given if no teammates were hurt or killed."
+L.help_karma_clean_half = [[
+When a player's Karma is above the starting level (meaning the Karma max has been configured to be higher than that), all their Karma increases will be reduced based on how far their Karma is above that starting level. So it goes up slower the higher it is.
+
+This reduction goes in a curve of exponential decay: initially it's fast, and it slows down as the increment gets smaller. This convar sets at what point the bonus has been halved (so the half-life). With the default value of 0.25, if a the starting amount of Karma is 1000 and the max 1500, and a player has Karma 1125 ((1500 - 1000) * 0.25 = 125), then his clean round bonus will be 30 / 2 = 15. So to make the bonus go down faster you’d set this convar lower, to make it go down slower you’d increase it towards 1.]]
+L.help_max_slots = "Sets the maximum amount of weapons per slot. '-1' means that there is no limit."
+
+L.label_killer_dna_range = "Max kill range to leave DNA"
+L.label_killer_dna_basetime = "Sample life base time"
+L.label_dna_scanner_slots = "DNA sample slots"
+L.label_dna_radar = "Enable classic DNA scanner mode"
+L.label_dna_radar_cooldown = "DNA scanner cooldown"
+L.label_radar_charge_time = "Recharge time after a radar sample"
+L.label_crowbar_shove_delay = "Cooldown after crowbar push"
+L.label_idle = "Enable idle mode"
+L.label_idle_limit = "Maximal idle time in seconds"
```

Comment:
As this time is the longest possible idle time its 'maximum' instead of 'maximal'. Had to google it, because maximal sounded wrong. 😄 
```suggestion
L.label_idle_limit = "Maximum idle time in seconds"
```